src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
In your link to the stylesheet, you have linked directly to a SCSS file. However, browsers can't interpret SCSS files. You need to compile your SCSS into CSS and then link to the compiled CSS file in your HTML. So, the href attribute should look like `href='./styles/index.css'`.